ui/components/Dashboard/ConnectionCharts.js

```diff
@@ -49,36 +50,45 @@ export default function ConnectionStatsChart({ classes }) {
     },
   };
 
-
-
   return (
     <div className={classes.dashboardSection}>
       <Link href="/management/connections">
         <Typography variant="h6" gutterBottom  className={classes.link}>
           Connections
         </Typography>
       </Link>
-      <Box  sx={{ display : "flex",justifyContent : "center",alignItems : "center",alignContent : "center",height : "100%" }}>
-        { chartData.length > 0 ?
-          <BBChart options={chartOptions} />
-          : (
-            <div
-              style={{
-                padding : "2rem",
-                display : "flex",
-                justifyContent : "center",
-                alignItems : "center",
-                flexDirection : "column",
-              }}
-            >
-              <Typography style={{ fontSize : "1.5rem", marginBottom : "1rem" }} align="center" >
+      <Box
+        sx={{
+          display : "flex",
+          justifyContent : "center",
+          alignItems : "center",
+          alignContent : "center",
+          height : "300px",  // Fixed height for consistency
```

Comment:
No need to add comments for simple things.

Reply:
oof, I forgot to remove them. I had them for my own track of things